fuuk/people/tests/test_views.py

```diff
@@ -377,13 +377,14 @@ def setUp(self):
         Person.objects.create(type='BC', first_name='Finished', last_name='Student', human=H5,
                               advisor=P1, is_active=False)
         # Courses
-        Course.objects.create(name='Testing course', code='AA001').lectors.add(P1)
+        self.C1 = Course.objects.create(name='Testing course', code='AA001')
+        self.C1.lectors.add(P1)
```

Comment:
Variables should contain only lowercase letters and underscores and their names should express their content.